cumulus_library/studies/core/builder_patient.py

```diff
@@ -18,9 +18,52 @@
 class PatientBuilder(BaseTableBuilder):
     display_text = "Creating Patient tables..."
 
+    @staticmethod
+    def make_extension_query(
+        schema: str,
+        cursor: databases.DatabaseCursor,
+        parser: databases.DatabaseParser,
+        name: str,
+        url: str,
+    ) -> str:
+        has_extensions = sql_utils.is_field_present(
+            schema=schema,
+            cursor=cursor,
+            parser=parser,
+            source_table="patient",
+            source_col="extension",
+            expected={
+                "extension": {
+                    "url": {},
+                    "valueCoding": [
+                        "code",
+                        "system",
+                    ],
+                },
+                "url": {},
+            },
+        )
+        if has_extensions:
+            config = sql_utils.ExtensionConfig(
+                source_table="patient",
+                source_id="id",
+                target_table=f"core__patient_ext_{name}",
+                target_col_prefix=name,
+                fhir_extension=url,
+                ext_systems=["ombCategory", "detailed"],
```

Comment:
Note: I dropped `text` from this list, because that uses `valueString`, not `valueCoding` so it would never yield anything anyway, since the jinja code doesn't check `valueString` -- open question as to whether we want to add support for that (`text` is the only actually required component of the extension after all, and all we care about is a display string... but that's for another PR).

Reply:
https://github.com/smart-on-fhir/cumulus-library/issues/222